### ISSUE TITLE
Mobile | Bootstrap | Calendar date pick removes modal

### ISSUE THREAD

**Issue by aviadhahami** [720kb/angular-datepicker#50]
When using the calendar inside a Bootstrap modal, picking a date inside the calendar closes the modal

**Comment by 45kb** [comment]
@aviadhahami  Hi thanks for reporting, is bootstrap UI that you are using or just bootstrap css?

Could you be so gentle to provide a plunkr or fiddle to show your code/ use case.

Thank you

**Comment by aviadhahami** [comment]
Hey ! 
So, issue was, as I suspected, that the modal is closed once there's a click outside of the modal window itself.
On mobile, the Calendar is larger than my specific modal, hence clicking on a date was interpreted as a click outside the modal therefor closing it.

The changes I made in order to fix this is changing the default values of the modal `backdrop` and `keyboard` to false instead of true.
[changes were made inside the original JS file of Bootstrap, `modal.js`]


Thanks anyhow :+1:
